test/TestServerProjectsLowLevel/paging/Generated/PagingClient.cs

```diff
@@ -95,10 +96,11 @@ async IAsyncEnumerable<Page<BinaryData>> CreateEnumerableAsync(string nextLink,
         /// <summary> A paging operation that must return result of the default &apos;value&apos; node. </summary>
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
         /// <remarks>
+        /// Below is the JSON schema for the request and response payloads.
```

Comment:
For paging operations, let's change the text here to say:

Below is the JSON schema for the request payload and for one item in the pageable response.